File: practica n6/autor.py
class Autor:
    def __init__(self, nombre: str, nacionalidad: str):
        self.nombre = nombre
        self.nacionalidad = nacionalidad


    def mostrar_info(self) -> None:
        print(f"Autor: {self.nombre}, Nacionalidad: {self.nacionalidad}")

    def __str__(self) -> str:
        return f"{self.nombre} ({self.nacionalidad})"

File: practicas/bibli/autor.java
package practicas.bibli;

public class autor {

    private String nombre;
    private String nacionalidad;

    public autor(String nombre, String nacionalidad) {
        this.nombre = nombre;
        this.nacionalidad = nacionalidad;
    }

    public void mostrarInfo() {
        System.out.println("Autor: " + nombre + ", Nacionalidad: " + nacionalidad);
    }
}
